Description: don't install stormlib
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-03-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/extern/StormLib/CMakeLists.txt
+++ b/extern/StormLib/CMakeLists.txt
@@ -9,7 +9,7 @@
 include(CMakeDependentOption)
 
 option(BUILD_SHARED_LIBS "Compile shared libraries" OFF)
-option(STORM_SKIP_INSTALL "Skip installing files" OFF)
+option(STORM_SKIP_INSTALL "Skip installing files" ON)
 option(STORM_USE_BUNDLED_LIBRARIES
     "Force use of bundled dependencies instead of system libraries."
     OFF
